internal/ir/xds.go

```diff
@@ -49,19 +49,19 @@ var (
 // +k8s:deepcopy-gen=true
 type Xds struct {
 	// AccessLog configuration for the gateway.
-	AccessLog *AccessLog
```

Comment:
needs `omitempty` tags as well since they show up in the o/p as `null` atm

Reply:
Done